packages/shared/src/components/RenderMarkdown.tsx

```diff
@@ -137,23 +141,46 @@ const RenderMarkdown = ({
             loadLanguages();
           }
 
-          return !inline && language ? (
-            <SyntaxHighlighterAsync
-              {...props}
-              customStyle={containerReset}
-              language={language}
-              useInlineStyles={false}
-              wrapLongLines
-              codeTagProps={{
-                className: codeClassName,
-              }}
-            >
-              {String(children).replace(replaceNewLineRegex, '')}
-            </SyntaxHighlighterAsync>
-          ) : (
-            <code {...props} className={codeClassName}>
-              {children}
-            </code>
+          return (
+            <>
+              {language && (
+                <div className="flex justify-between py-3 px-6 bg-theme-active align-center">
```

Comment:
Probably best to add text-teriary here as well to ensure color of the language
And not sure if it's a different size

Reply:
Think also the padding is off (seems to big)